lti_consumer/lti_xblock.py

```diff
@@ -1427,6 +1427,8 @@ def _get_modal_position_offset(self, viewport_percentage):
         Returns:
             float: The css position offset to apply to the modal window
         """
+        if not viewport_percentage:
+            viewport_percentage = 80        # set to default value in case viewport_percentage is None
```

Comment:
is there a way to get this value from the default value of the instance?

Reply:
Yes that can be done using this

`LtiConsumerXBlock.modal_height.default`

but the problem is that the method is being called for `modal_width` as well. At the time, both have the default value of `80` but if one were to change, we would have to have different default value conditions for both in this method.

Reply:
Ok, I am ok with it. 👍